app/frontend/src/components/Divider.tsx

```diff
@@ -1,16 +1,38 @@
 import { Divider as MuiDivider } from "@material-ui/core";
+import classNames from "classnames";
 import React from "react";
 import makeStyles from "utils/makeStyles";
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    marginBottom: theme.spacing(1),
-    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(2),
+    marginTop: theme.spacing(2),
+  },
+
+  marginTopExtra1: {
+    marginTop: theme.spacing(3),
+  },
+
+  marginBottomExtra1: {
+    marginBottom: theme.spacing(3),
   },
 }));
 
-export default function Divider() {
+export interface DividerProps {
+  margBottom?: boolean;
```

Comment:
If there were no other changes I wouldn't have mentioned, but can this just be `marginTop` and `marginBottom`

Reply:
I didn't want it to have the exact same name as the css property to avoid confussion.

Reply:
I did the following: as I had to set another prop for margins in barWithHelp, I used the same syntax (borrowed from Tailwind), so here the props are `mt3` and `mb3` (they stand for margin-top-3 and margin-bottom-3) and in barWithHelp the prop will be `mb2`.